### ISSUE TITLE
Menu - make drill down menu respect prefers-reduced-motion

### ISSUE THREAD

**Issue by srambach** [patternfly/patternfly#6739]
**Describe the feature**
The drill-down menu currently uses a slide to show the drilled-in menus. This is potentially problematic and should have a `prefers-reduced-motion` transition that uses opacity (probably) instead of translateX.
